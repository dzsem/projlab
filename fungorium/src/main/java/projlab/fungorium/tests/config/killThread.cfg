# Kill Thread
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD THREADKILLINGTECTON
SET TECTONKILLCHANCE 100
# 1
ADD MUSHROOMTHREAD 0 0