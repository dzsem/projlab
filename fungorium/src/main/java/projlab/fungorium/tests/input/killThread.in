# Kill Thread
# ------------------------------------------
# Input
# ------------------------------------------
# Tecton kills thread on the end of the round
NEXTROUND
# Print states
# ------------------------------------------
LIST THREADKILLINGTECTON
LIST MUSHROOMTHREAD