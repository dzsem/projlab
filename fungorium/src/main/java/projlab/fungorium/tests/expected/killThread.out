THREADKILLINGTECTON 0 0 0 0